docs/en/ingest-management/agent-policies.asciidoc

```diff
@@ -0,0 +1,215 @@
+and agent monitoring status can be changed as necessary:
+
+. In {fleet}, click *Policies*.
+Click on the name of the policy you want to edit or delete.
+
+. Click the *Settings* tab, make changes, and click *Save changes*
++
+Alternatively, click *Delete policy* to delete the policy.
+Existing data will not be deleted.
+Any agents assigned to a policy must be unenrolled before a policy can be deleted.
+
+[discrete]
+[[integration-updates]]
+== Integration updates
+
+Elastic releases integration updates periodically.
+Integrations are delivered to the Elastic Stack, via the online Elastic Package Registry
+and the {kib} Elastic Package Manager. When a new Integration is released by Elastic,
+it will show up in the Integrations listing in {kib} after a restart (like during a stack upgrade).
+// to do: link to the Fleet API docs
+A refresh can also be triggered manually, with the Fleet API.
+
+NOTE: The latest, shipped Elastic Package registry version of an integration is the only version
+that can be integrated into {agent} policies. Previously installed versions will continue to work.
+
+[discrete]
+[[update-an-integration]]
+=== Update an integration
+
+. In {fleet}, click *Integrations*.
+Search for and select the integration you'd like to update.
+
+. If an update is available, click *Update*.
++
+Because updates can change the behavior of hosts,
+they must be explicitly request in the {fleet} app.
+
+. <<create-a-policy,Create a new policy>>.
+
+. <<add-integration,Add the integration to the policy>>.
+The newer version will automatically be used
+
+. <<apply-a-policy,Apply the policy>> to an {agent}.
++
+TIP: In larger deployments, integration updates should be tested on a sample {agent}
+before rolling out a larger upgrade initiative.
+Only after a small trial is deemed successful should the updated policy be
+<<roll-out-an-integration,rolled out all hosts>>.
+
+[discrete]
+[[roll-out-an-integration]]
+=== Roll-out an integration update
+
+After successfully testing an integration update,
+it can be safely rolled-out to additional hosts:
+
+. In {fleet}, click *Policies*.
+Click on the name of the policy you want to edit or delete.
+
+. Search or scroll to a specific integration.
+Open the *Actions* menu and select *Delete integration*.
+
+. Click *Add integration* and re-add the freshly deleted integration.
+The updated version will be used and applied to all {agent}s.
+
+. Repeat this process for each policy with the out-of-date integration.
+
+NOTE: In some instances, for example, when there are hundreds or thousands of different {agent}s and
+policies that need to be updated, this upgrade path is not feasible.
+In this case, update one policy and use the <<copy-policy>> action to apply the updated policy versions to additional policies.
+The downside of this method is losing
+the granularity of assessing the individual Integration version changes individually across {policies}.
```

Comment:
Should the attribute be {policy}'s? Looks like the attribute could be used in a few other sections.

Reply:
```suggestion
the granularity of assessing the individual Integration version changes individually across policies.
```